### PR TITLE
Implemented cube size configurator.

### PR DESCRIPTION
**Pull request by naveencl**
please check.

### CODE REVIEW

File: wok/p5/sketch.js [hunk 1]
```diff
@@ -1,84 +1,132 @@
...
+    grid = new Grid(
+      parseInt(xInput.value()),
+      parseInt(yInput.value()),
+      parseInt(zInput.value())
+    );
+    for (let key of tiles) {
+      const c = key.split("|").map(i => +i);
+      grid.get(grid.toIndex(...c)).type = map.tiles[key];
+    }
+    for (let key in food) {
```
Comment (mraak) [inline wok/p5/sketch.js]: What is this food here?

**Comment by mraak** [comment]
@naveencl since we're dealing with graphic interfaces a screenshot or two would be nice next to the pull request.

File: wok/p5/sketch.js (hunk 1 above)
Comment (naveencl) [reply wok/p5/sketch.js]: "food" was already used in the code. We reused the existing code. Miran said, "food is part of the maze code, it's used as a reward for path finder".

Comment (mraak) [reply wok/p5/sketch.js]: There's no need for food here. The path will be just random walk.

Comment (naveencl) [reply wok/p5/sketch.js]: Removed food,

Comment (naveencl) [reply wok/p5/sketch.js]: image

**Pull request merged**